packages/vuetify/src/components/VDatePicker/util/determineWeeknumber.ts

```diff
@@ -0,0 +1,58 @@
+const hourInMilliseconds = 60 * 1000
+const dayInMilliseconds = 24 * 60 * hourInMilliseconds
```

Comment:
`millisecondsInHour`, `millisecondsInDay`

Reply:
Refactored dayOfYear method without the use of the constants so removed them